packages/flutter/lib/src/material/text_field.dart

```diff
@@ -92,6 +92,26 @@ class TextField extends StatefulWidget {
   /// [maxLengthEnforced], [scrollPadding], [maxLines], [maxLength],
   /// and [enableInteractiveSelection] arguments must not be null.
   ///
+  /// ## Sample code
+  ///
+  /// This example shows how to create [TextField] as password input which
+  /// allows 25 characters only and calls callback method when user presses the
+  /// 'done' button on the keyboard.
+  ///
+  /// ```dart
+  /// void _login(String passwd) {
+  ///   // provide login logic
+  /// }
+  ///
```

Comment:
You should add a 
```dart
/// ...
///
```
here, just to indicate that these two statements are probably separated by some logic in a real app.